src/quo2/core.cljs

```diff
@@ -17,6 +17,7 @@
     quo2.components.buttons.wallet-button.view
     quo2.components.buttons.wallet-ctas.view
     quo2.components.browser.browser-input.view
+    quo2.components.browser.dApp.item.view
```

Comment:
I'm not sure about the name of this component with pascal case because I didn't see any other example with such naming, can you think of a better name?

Reply:
imo make it lower case, i.e ` quo2.components.browser.dapp.item.view`